modules/articles/client/config/articles.client.routes.js

```diff
@@ -12,25 +16,56 @@ angular.module('articles').config(['$stateProvider',
       })
       .state('articles.list', {
         url: '',
-        templateUrl: 'modules/articles/client/views/list-articles.client.view.html'
+        templateUrl: 'modules/articles/client/views/list-articles.client.view.html',
+        controller: 'ArticlesListController',
+        controllerAs: 'vm'
       })
       .state('articles.create', {
         url: '/create',
-        templateUrl: 'modules/articles/client/views/create-article.client.view.html',
+        templateUrl: 'modules/articles/client/views/form-article.client.view.html',
+        controller: 'ArticlesController',
+        controllerAs: 'vm',
+        resolve: {
+          articleResolve: newArticle
```

Comment:
One additional benefit is to share the resolved value among nested views.  I use nested views all the time, so if I pass the Article in the resolve then populate it in one controller that value gets shared among my other views and controllers.  If you instantiate the model in the controller the nested views would not get that data.  I think we should start looking at nested views for much of the project and this is a good start.